File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int number_blocks;
    do
    {
        number_blocks = get_int("Size: ");
    }
    while (number_blocks < 1 || number_blocks > 8);

      for (int i = 0; i < number_blocks; i++)
      {
         for (int j = 0; j < number_blocks; j++)
        {
          if (j < (number_blocks - i - 1))
          {
              printf(" ");
          }
          else
          {
              printf("#");
          }
        }
      printf("\n");
      }
}

File: pset6/mario.py
import cs50 

while True:
    number_blocks = cs50.get_int("Height: ")
    
    if number_blocks >= 1 and number_blocks <= 8:
        break
    
for i in range(number_blocks):
    for j in range(number_blocks):
        if i + j >= number_blocks - 1:
            print("#", end = "")
        else:
            print(" ", end = "")

    print("")
